app/assets/stylesheets/main.scss

```diff
@@ -25,7 +25,7 @@
 }
 
 .assignment_summary {
-  @include border-radius($radius);
+  border-radius: $radius;
```

Comment:
Properties should be ordered background-color, border, border-radius, display, margin, padding, text-align, width